packages/apps/frontera/src/routes/flow-editor/src/Header.tsx

```diff
@@ -145,10 +145,8 @@ export const Header = observer(
                 <>
                   <ChevronRight className='text-gray-400' />
                   <span className='font-medium cursor-default'>
-                    {`${flow?.value?.participants?.length} ${
-                      flow?.value?.participants?.length === 1
-                        ? 'Contact'
-                        : 'Contacts'
+                    {`${store.ui.searchCount} ${
+                      store.ui.searchCount === 1 ? 'Contact' : 'Contacts'
```

Comment:
searchCount is used for something else. The source of truth for this count needs to be flow.value.participants.length.

Reply:
@acalinica the participants.length includes the archived contacts

Reply:
That still does not justify using store.ui.searchCount for this. Store.ui.searchCount can be set from many other places in the app that are unrelated with this logic, hence it's not safe to use it like this.

Reply:
just filter the participants to not include the one that are archived

Reply:
The right thing to do here is to change the query that returns this count to not sum up the archived ones.

Reply:
Or just filter the participants that are archived and then get the length.